src/test/scala/chiselTests/QueueSpec.scala

```diff
@@ -233,4 +233,35 @@ class QueueSpec extends ChiselPropSpec {
       }
     }
   }
+
+  property("Queue.irrevocable should elaborate") {
```

Comment:
Since this test is really just doing elaboration and there's no reason to check the functionality of an irrevocable `Queue`, how about stripping this down to a minimal test that elaboration works?

Something like the following is how I would do it in the Stage/Phase world:

```scala
  import chisel3.stage.phases.Elaborate
  import chisel3.stage.ChiselGeneratorAnnotation
  import chisel3.experimental.MultiIOModule

  property("Queue.irrevocable should elaborate") {
    class IrrevocableQueue extends MultiIOModule {
      val in = Wire(Decoupled(Bool()))
      val iQueue = Queue.irrevocable(in, 1)
    }
    (new Elaborate).transform(Seq(ChiselGeneratorAnnotation( () => new IrrevocableQueue )))
  }
```